design-documents/catalog-domain/understanding-of-complex-product-types.md

```diff
@@ -0,0 +1,33 @@
+# Understanding of complex product types
+
+## 1. Definitions
+See [Simple Products](simple-products.md)
+* Complex product is a virtual representation of multiple real SKUs at the storefront.
+* Currently, Magento does support three complex product types - Bundle, Grouped and Configurable.
+* For the simplicity and data consistency a complex product has SKU.
+In fact this SKU is virtual.
+* Complex product may have all attributes that required to render product details page. 
+* Complex product does not have attributes that describe physical product characteristics.
+
+## 2. Inventory Management
+
+* Complex products are uncountable.
+* If *IMS* is synchronized with Catalog a complex products may have stock status.
+
+## 3. Product prices
+* Complex product does not have itself price.
```

Comment:
It would be good to have currently supported scenarios described with their equivalent in the new system.